zulip/integrations/bridge_with_matrix/README.md

```diff
@@ -27,20 +26,37 @@ details mentioned below. For example:
 * If you are running from the Zulip GitHub repo: `python matrix_bridge.py --write-sample-config matrix_bridge.conf`
 
 ### 1. Zulip endpoint
-1. Create a generic Zulip bot, with a full name like `IRC Bot` or `Matrix Bot`.
-2. Subscribe the bot user to the stream you'd like to bridge your IRC or Matrix
-   channel into.
+1. Create a generic Zulip bot, with a full name such as `Matrix Bot`.
+2. The bot is able to subscribe to the necessary streams itself if they are
+   public. (It may even create them if they do not exist and if the bot user
```

Comment:
What if the user somehow made a typo on the stream name, and the typo stream accidentally gets created?

Reply:
Then need to remove the wrong stream and fix the typo... Or do you think that this feature is not really useful? :thinking:

Reply:
There is no way to delete a stream from the web admin UI. Possible to do so via DB op but tedious, and so the mistake is expensive, IMO.

Reply:
Hm... But in the "Stream Settings" you can "delete" ("archive") a stream... And the stream would be empty, so this would not be too bad? But I understand your point...

Reply:
@timabbott thoughts?

Reply:
I think this is my only remaining concern.

Reply:
I still think it is dangerous to accidentally auto-create a stream, where it can only be archived later if the stream creation is a mistake.

Reply:
Ok, I'll change it :)

Reply:
Ok, I'll change it :)

Reply:
Has this been changed? If so, the documentation here is outdated because it still says auto-subscribe.

Reply:
sorry, done